db/schema.rb

```diff
@@ -62,6 +70,64 @@
 
   add_foreign_key "application_version", "application", name: "application_version_application_id_fkey"
 
+  create_view "geography_columns", sql_definition: <<-SQL
```

Comment:
not too sure why these are included here for this change and why we would need a view of Postgis extension tables?

Reply:
No idea, this was auto generated when I did the migration.

Reply:
So 2 things have happened. First, we enabled postgis so that LocationService can figure out if a postcode is inside the M25 (because this now happens here rather than in assess). Second, one of our Rails dependencies is now struggling with postgis, because it started doing this on assess recently to - when the schema file is regenerated it's seeing some postgis tables and thinking they're manually created tables that belong in the schema file. (Which then messes up when you try to use the schema to run things like `rails db:prepare`